Remove attempts add the -O compiler option -- this ends up lowering optimization
level for most people (from -O2 to -O).

	-mi

--- configure	2015-05-12 14:46:37 UTC
+++ configure	2015-09-22 22:15:58 UTC
@@ -447,5 +447,5 @@
 # Configure compiler to use in further tests
 
-cc="$bytecc -O $bytecclinkopts"
+cc="$bytecc $bytecclinkopts"
 export cc cclibs verbose
 
@@ -1687,10 +1687,4 @@
 fi
 
-# Add Unix-style optimization flag
-bytecccompopts="-O $bytecccompopts"
-dllcccompopts="-O $dllcccompopts"
-nativecccompopts="-O $nativecccompopts"
-sharedcccompopts="-O $sharedcccompopts"
-
 # Final twiddling of compiler options to work around known bugs
 
